contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -60,13 +59,15 @@ import (
 	"k8s.io/kubernetes/contrib/mesos/pkg/scheduler/podtask"
 	mresource "k8s.io/kubernetes/contrib/mesos/pkg/scheduler/resource"
 	"k8s.io/kubernetes/contrib/mesos/pkg/scheduler/uid"
-	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/resource"
 	client "k8s.io/kubernetes/pkg/client/unversioned"
 	clientauth "k8s.io/kubernetes/pkg/client/unversioned/auth"
 	"k8s.io/kubernetes/pkg/master/ports"
 	etcdstorage "k8s.io/kubernetes/pkg/storage/etcd"
 	"k8s.io/kubernetes/pkg/tools"
+
+	// lock to this API version, compilation will fail when this becomes unsupported
+	_ "k8s.io/kubernetes/pkg/api/v1"
```

Comment:
should probably invoke an empty/noop func in v1 to really force the compiler to pick up on this (endpoints controller used to do something similar with the old v1beta1, v1beta2)